ql/ql/src/codeql_ql/ast/internal/AstNodes.qll

```diff
@@ -60,7 +60,7 @@ newtype TAstNode =
   TPredicateExpr(QL::PredicateExpr pe) or
   TAnnotation(QL::Annotation annot) or
   TAnnotationArg(QL::AnnotArg arg) or
-  TYamlCommemt(Yaml::Comment yc) or
+  TYamlComment(Yaml::Comment yc) or
```

Comment:
Hopefully not a problem

Reply:
Not a problem.  
We don't need to follow any kind of deprecation policy inside the `ql/` folder, as it's not part of any product.  

We should be good as long as the CI passes.